### PR TITLE
Adding Sardinian strings

### PR DESCRIPTION
**Pull request by asereze**
Feature: it adds a Sardinian translation  (name of the language in Sardinian: "Sardu", ISO 639-1 code: "sc")

### CODE REVIEW

File: openlibrary/i18n/sc/messages.po [hunk 1]
```diff
@@ -0,0 +1,5761 @@
...
+"risposta luego. Nos podes fintzas <a href=\"https://twitter.com/"
+"openlibrary\">chircare in Twitter</a>."
+
+#: support.html:19
+msgid ""
+"Please check our <a href=\"/help/\">Help Pages</a> and <a href=\"/help/"
+"faq\">Frequently Asked Questions</a> (FAQ) to see if your question is answered "
+"there. Thank you."
+msgstr ""
+"Pro praghere abbista·ti sas <a href=\"/help/\">Pàginas de agiudu/a> e sas <a "
```
Comment (cdrini) [inline openlibrary/i18n/sc/messages.po:539]:
```suggestion
"Pro praghere abbista·ti sas <a href=\"/help/\">Pàginas de agiudu</a> e sas <a "
```

**Review by cdrini — COMMENTED** [review]
Awesome, thank you @asereze ! I found one issue with the html, but otherwise things look good!

**Comment by asereze** [comment]
> Awesome, thank you @asereze ! Our tests found one issue with the html, but otherwise things look good!

Oh, I didn't notice that. Thank you!

**Comment by cdrini** [comment]
Thank you so much @asereze ! Sardinian users will, after our next deploy next week, automatically have the website appear in Sardinian :) I've also created a new issue for adding Sardinian to our list of languages so anyone can select the language: #8857

**Comment by scottbarnes** [comment]
I just wanted to chime in also to say that this is great, @asereze. Thanks so much!

**Comment by asereze** [comment]
@cdrini @scottbarnes You are welcome! Thank you very much to all of you for your great work!

**Pull request merged**